src/gr/polynomial.c

```diff
@@ -360,12 +360,60 @@ polynomial_neg(gr_poly_t res, const gr_poly_t mat, gr_ctx_t ctx)
     return gr_poly_neg(res, mat, POLYNOMIAL_ELEM_CTX(ctx));
 }
 
+int
+polynomial_add_ui(gr_poly_t res, const gr_poly_t poly, ulong c, gr_ctx_t ctx)
+{
+    return gr_poly_add_ui(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_add_si(gr_poly_t res, const gr_poly_t poly, slong c, gr_ctx_t ctx)
+{
+    return gr_poly_add_si(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_add_fmpz(gr_poly_t res, const gr_poly_t poly, const fmpz_t c, gr_ctx_t ctx)
+{
+    return gr_poly_add_fmpz(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_add_fmpq(gr_poly_t res, const gr_poly_t poly, fmpq_t c, gr_ctx_t ctx)
+{
+    return gr_poly_add_fmpq(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
 int
 polynomial_add(gr_poly_t res, const gr_poly_t poly1, const gr_poly_t poly2, gr_ctx_t ctx)
 {
     return gr_poly_add(res, poly1, poly2, POLYNOMIAL_ELEM_CTX(ctx));
 }
 
+int
+polynomial_sub_ui(gr_poly_t res, const gr_poly_t poly, ulong c, gr_ctx_t ctx)
+{
+    return gr_poly_sub_ui(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_sub_si(gr_poly_t res, const gr_poly_t poly, slong c, gr_ctx_t ctx)
+{
+    return gr_poly_sub_si(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_sub_fmpz(gr_poly_t res, const gr_poly_t poly, const fmpz_t c, gr_ctx_t ctx)
+{
+    return gr_poly_sub_fmpz(res, poly, c, POLYNOMIAL_ELEM_CTX(ctx));
+}
+
+int
+polynomial_sub_fmpq(gr_poly_t res, const gr_poly_t poly, fmpq_t c, gr_ctx_t ctx)
```

Comment:
Missing const

Reply:
This mistake was copied from `mul`, so I'll fix it there too.